catalogs/harvard.yaml

```diff
@@ -6,28 +6,14 @@ sources:
   ihp:
     driver: oai_xml
     args:
-      collection_url: https://api.lib.harvard.edu/oai
-      metadata_prefix: oai_dc
-      set: "ihp"
+      collection_url: https://api.lib.harvard.edu/oai?verb=ListRecords&metadata_prefix=mods&set=ihp
```

Comment:
The build failure you're seeing is due to moving the `metadata_prefix` up into the URL. This should be an argument (required by the driver), as well as the `set`

Apologies, I should have mentioned that would be necessary when using `oai_xml`

Reply:
Oops, too many changes. One minute